R/prophet.R

```diff
@@ -748,7 +752,7 @@ Prophet <- function(jaspResults, dataset = NULL, options) {
 }
 
 .prophetCreateForecastPlots <- function(jaspResults, dataset, options, ready) {
-  if (!ready) return()
+  if (!ready || is.null(jaspResults[["prophetResults"]][["object"]][["prophetPredictionResults"]])) return()
```

Comment:
Hmm, this means that people may click on a forecast plot, expecting an output, but they won't see anything... 

How about instead creating the container and setting an informative message that explains what's happening:

```r
if((options[["forecastPlotsOverall"]] || options [["forecastPlotsTrend"]]) && is.null(prophetPredictionResults) {
  prophetForecastPlots$setError(gettext("Cannot draw forecast plots; no forecasts computed."))
  return()
}
```

Reply:
Can this be solved asap? We'd like really much to release tomorrow....

Reply:
Thanks for the suggestion! I tried it but it seems like the setError function only works on tables and plot objects, not containers themselves https://github.com/jasp-stats/INTERNAL-jasp/issues/823.

Since using the stop function would make the analysis run again and the computation of prophet can be quite time intensive, I now added an empty table that I use the setError function on. 
Let me know if this is too hacky of a solution.

Reply:
Yes, that's ok!